<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_matricula">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: .5px solid
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Matricula</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEliminar_alumno">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color:rgb(205, 187, 227);
	border-radius:10px;
	border:10px;
}


QPushButton:hover{
background-color: rgb(170, 255, 127);
	border-radius:10px;
	border: 10px;
}</string>
    </property>
    <property name="text">
     <string>ELIMINAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLimpiar_alumno">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>490</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color:rgb(205, 187, 227);
	border-radius:10px;
	border:10px;
}


QPushButton:hover{
background-color: rgb(170, 255, 127);
	border-radius:10px;
	border: 10px;
}</string>
    </property>
    <property name="text">
     <string>LIMPIAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe MDL2 Assets</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eliminar Alumno</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRegresar_alumno">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>490</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color:rgb(205, 187, 227);
	border-radius:10px;
	border:10px;
}


QPushButton:hover{
background-color: rgb(170, 255, 127);
	border-radius:10px;
	border: 10px;
}</string>
    </property>
    <property name="text">
     <string>REGRESAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>491</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagenes/Logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_contrasena">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: .5px solid
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_carrera">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: .5px solid
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_apellidos">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: .5px solid
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apellidos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nombre">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: .5px solid
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_correo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: .5px solid
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nombre">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Carrera</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Correo Electrónico</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbConsultar_alumno">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color:rgb(205, 187, 227);
	border-radius:10px;
	border:10px;
}


QPushButton:hover{
background-color: rgb(170, 255, 127);
	border-radius:10px;
	border: 10px;
}</string>
    </property>
    <property name="text">
     <string>CONSULTAR</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_matricula</tabstop>
  <tabstop>pbEliminar_alumno</tabstop>
  <tabstop>pbLimpiar_alumno</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
